pandas/core/indexes/base.py

```diff
@@ -6067,6 +6074,11 @@ def map(self, mapper, na_action: Literal["ignore"] | None = None):
             If the function returns a tuple with more than one element
             a MultiIndex will be returned.
 
+        See Also
+        --------
+        Index.apply : Invoke function on values of Series.
```

Comment:
It doesn't appear these methods exist on `Index`

Reply:
Oh, sorry about that. Updating the methods!